bundle/bundle.go

```diff
@@ -2,10 +2,30 @@ package bundle
 
 import (
 	"embed"
+	"sync"
+
+	"github.com/open-policy-agent/opa/bundle"
+
+	rio "github.com/styrainc/regal/internal/io"
 )
 
 // Bundle FS will include the tests as well, but since that has negligible impact on the size of the binary,
 // it's preferable to filter them out from the bundle than to e.g. create a separate directory for tests
 //
 //go:embed *
 var Bundle embed.FS
+
+//nolint:gochecknoglobals
+var (
+	loadedBundle     bundle.Bundle
+	loadedBundleOnce sync.Once
+)
+
+// LoadedBundle returns the loaded Regal bundle for this version of Regal.
```

Comment:
I take it you'd like to keep it being lazily loaded? Because if it was going to be used anyways, often times sync.Once etc can be replaced with a package var, like
```go
var loadedBundle = io.MustLoadRegalBundleFS(Bundle)
```

Reply:
Perhaps we should just do this 👍 I was remembering a discussion about how inits in the ls implementation were slowing down the linting cmd, but that's not the case here.

Reply:
Fixed in 9c2e409614028150e2ac19835d89466fccc006a1